resources/examples/manager/BasicAssetLibrary/tests/resources/library_apiComplianceSuite.json

```diff
@@ -1,5 +1,4 @@
 {
-  "$schema": "https://raw.githubusercontent.com/TheFoundryVisionmongers/OpenAssetIO/main/resources/examples/manager/BasicAssetLibrary/schema.json",
```

Comment:
[consider][minor] It's unfortunate to remove it, since it does work (at least in JetBrains IDEs) - it's just that `$schema` gets flagged as not being in the schema. 

Selfishly, I can also configure the schema for a JSON file through the settings in JetBrains IDEs, so removing it from the .json itself doesn't preclude nice schema validation for me, it's just less convenient.

Is it just that we don't want to go down the route of using non-standard extensions?

Reply:
Happy to leave it in here if you like, I guess I was just less keen on adding it to the schema itself as that seems very atypical?

Reply:
So just let it flag a warning in IDEs that don't treat `$schema` as a special case?  I think I prefer that, rather than omit it.

Reply:
I've dropped the commit that removed it from the library `json`.